docs/cloud-workload-security/agent_expressions.md

```diff
@@ -133,6 +133,7 @@ The *file.rights* attribute can now be used in addition to *file.mode*. *file.mo
 | -------- | ---- | ---------- |
 | `container.id` | string | ID of the container |
 | `container.tags` | string | Tags of the container |
+| `process.ancestors.arg0` | string |  |
```

Comment:
@safchain 👋 Thanks for opening this PR! It looks like the descriptions are empty here. Do you plan on adding descriptions?

Reply:
@sarina-dd thanks, my bad I forgot to commit a file, I updated the PR